### PR TITLE
Sfmc first refresh release

### PR DESCRIPTION
**Pull request by peterpogorski**

### CODE REVIEW

File: sfmc_release_notes/SF_managed_clusters_first_refresh_release_notes.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+* [Virtual Machine Scale Set Managed Identities](https://docs.microsoft.com/azure/service-fabric/how-to-managed-identity-managed-cluster-virtual-machine-scale-sets) - Each node type in a Service Fabric managed cluster is backed by a Virtual Machine Scale Set. To enable managed identities to be used with a managed cluster node type, a property vmManagedIdentity has been added to node type definitions. This is a list that contains user defined identities such as Azure userAssignedIdenties.  
+* [NSG rule configuration](https://docs.microsoft.com/en-us/azure/service-fabric/how-to-managed-cluster-configuration#networking-configurations) - You can now customize Network Security Group rules enabling you to meet any compliance requirements for your scenarios.
+
+## Known Issues
+
+### ARM application deployments
+
+* Stateless service instance count cannot be updated after initial deployment.
+* When redeploying an application, the services must first be deleted through ARM.
+* When using the PowerShell cmdlets, the GET application command may fail.
```
Comment (masnider) [inline sfmc_release_notes/SF_managed_clusters_first_refresh_release_notes.md:33]: Please describe what the user should do in such a scenario.

File: sfmc_release_notes/SF_managed_clusters_first_refresh_release_notes.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Microsoft Azure Service Fabric managed clusters first refresh release (Preview)
```
Comment (masnider) [inline sfmc_release_notes/SF_managed_clusters_first_refresh_release_notes.md:1]: Rather than "first refresh" can we call it by the version number?

Comment (masnider) [reply sfmc_release_notes/SF_managed_clusters_first_refresh_release_notes.md:1]: Ex: the 2021-01-01-preview release?

File: sfmc_release_notes/SF_managed_clusters_first_refresh_release_notes.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+* [Virtual Machine Scale Set Managed Identities](https://docs.microsoft.com/azure/service-fabric/how-to-managed-identity-managed-cluster-virtual-machine-scale-sets) - Each node type in a Service Fabric managed cluster is backed by a Virtual Machine Scale Set. To enable managed identities to be used with a managed cluster node type, a property vmManagedIdentity has been added to node type definitions. This is a list that contains user defined identities such as Azure userAssignedIdenties.  
+* [NSG rule configuration](https://docs.microsoft.com/en-us/azure/service-fabric/how-to-managed-cluster-configuration#networking-configurations) - You can now customize Network Security Group rules enabling you to meet any compliance requirements for your scenarios.
+
+## Known Issues
+
```
Comment (masnider) [inline sfmc_release_notes/SF_managed_clusters_first_refresh_release_notes.md:28]: Can we please add headers for the other feature areas and say "no known issues" ?

File: sfmc_release_notes/SF_managed_clusters_first_refresh_release_notes.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+### ARM application deployments
+
+* Stateless service instance count cannot be updated after initial deployment.
+* When redeploying an application, the services must first be deleted through ARM.
+* When using the PowerShell cmdlets, the GET application command may fail.
+
+## Upcoming Features
```
Comment (masnider) [inline sfmc_release_notes/SF_managed_clusters_first_refresh_release_notes.md:35]: Can we indicate any expected timeline for these, or when we will have the next update?

**Review by masnider — CHANGES_REQUESTED** [review]

**Review by linmeng08 — APPROVED** [review]

**Review by athinanthny — APPROVED** [review]

**Review by peterpogorski — COMMENTED** [review]
Fixes addressed

**Pull request merged**